graph/ops_convolve_test.go

```diff
@@ -240,3 +240,29 @@ func TestGradientConvolve(t *testing.T) {
 			tensors.FromScalarAndDimensions(0.0, 2, 2, 3, 6).Value(),
 		})
 }
+
+func TestConvolveForwardPass(t *testing.T) {
```

Comment:
What about "TestConvolveWithGroupCount" ? (Since `TestConvolve` is also "ForwardPass" so to say ?